{"name": "test_user_can_login", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "browser = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"be3d79baf73a632cf2c43a0d8d946374\")>\n\n    @allure.story(\"Check valid user can login\")\n    def test_user_can_login(browser):\n        with allure.step('Open login page'):\n            main_page = MainPage(browser)\n            main_page.open_main_page()\n            main_page.open_login_page()\n        with allure.step('Check login page displays'):\n            login_page = LoginPage(browser)\n            login_page.should_be_login_page()\n        with allure.step('Login as a user \"victoria.lisouskaya@gmail.com\"'):\n            login_page.login(\"victoria.lisouskaya@gmail.com\", \"111222\")\n        with allure.step('Check account page displays'):\n            account_page = AccountPage(browser)\n            account_page.should_be_account_page()\n        sign_up_final_page = SignUpFinalPage(browser)\n        sign_up_final_page.navigate_to_passwd_reset_page(\"https://www.torontocaterers.ca/user/dashboard\")\n        with allure.step('Check user profile page displays'):\n            user_profile_page = UserProfile(browser)\n>           user_profile_page.should_be_user_profile_page(\"test user\")\n\ntests\\test_username.py:29: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <pages.profile_page.UserProfile object at 0x01CFF0A0>, full_name = 'test user'\n\n    def should_be_user_profile_page(self, full_name: str):\n       # submit_btn = self.find_element(UserProfileLocators.LOCATOR_SUBMIT_BTN)\n       # submit_btn.click()\n    \n        user_name_lbl = self.find_element(UserProfileLocators.LOCATOR_USER_NAME_LBL)\n        print(\"!!!!!: fullname is:\" + full_name)\n        print(\"!!!!!: Text is:\" + user_name_lbl.text)\n>       assert full_name in user_name_lbl.text\nE       AssertionError\n\npages\\profile_page.py:14: AssertionError"}, "steps": [{"name": "Open login page", "status": "passed", "start": 1633274903495, "stop": 1633274906692}, {"name": "Check login page displays", "status": "passed", "start": 1633274906692, "stop": 1633274906712}, {"name": "Login as a user \"victoria.lisouskaya@gmail.com\"", "status": "passed", "start": 1633274906712, "stop": 1633274906973}, {"name": "Check account page displays", "status": "passed", "start": 1633274906973, "stop": 1633274907736}, {"name": "Check user profile page displays", "status": "failed", "statusDetails": {"message": "AssertionError\n", "trace": "  File \"C:\\Users\\viktorial\\Documents\\python-project\\final-project-new\\tests\\test_username.py\", line 29, in test_user_can_login\n    user_profile_page.should_be_user_profile_page(\"test user\")\n  File \"C:\\Users\\viktorial\\Documents\\python-project\\final-project-new\\pages\\profile_page.py\", line 14, in should_be_user_profile_page\n    assert full_name in user_name_lbl.text\n"}, "start": 1633274909186, "stop": 1633274909434}], "attachments": [{"name": "stdout", "source": "2cd74ab7-9049-44c9-8cc8-b05ec4a72369-attachment.txt", "type": "text/plain"}], "start": 1633274903494, "stop": 1633274909435, "uuid": "38e55415-4909-412f-9f60-20a669c8811c", "historyId": "a5c8ace36cac5ca1f746287da0869365", "testCaseId": "e4ca2f7627781931ea9f0bc7fd0a5329", "fullName": "tests.test_username#test_user_can_login", "labels": [{"name": "story", "value": "Check valid user can login"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_username"}, {"name": "host", "value": "viktorial"}, {"name": "thread", "value": "28176-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_username"}]}